cpapp_backend/cpapp/services/session_manager.py

```
@@ -0,0 +1,123 @@
+import uuid
+import logging
+from typing import Dict, Any, Optional
+from datetime import datetime
+from langchain_core.messages import HumanMessage, AIMessage
```

Ruff (0.12.2)

`langchain_core.messages.HumanMessage` imported but unused

Remove unused import

(F401)